docs/extend/extend-cli/cli-extending-a-plugin.md

```diff
@@ -93,91 +102,114 @@ const session = new Session({ hostname: "jsonplaceholder.typicode.com"});
 
 ### Creating a command definition
 
-Within Zowe CLI, the full command that we want to create is `zowe zowe-cli-sample list typicode-todos`. Navigate to `zowe-cli-sample-plugin/src/cli/list` and create a folder
-`typicode-todos`. Within this folder, create `TypicodeTodos.definition.ts`. Its content should be as follows:
-```typescript
-import { ICommandDefinition } from "@zowe/imperative";
-export const TypicodeTodosDefinition: ICommandDefinition = {
-    name: "typicode-todos",
-    aliases: ["td"],
-    summary: "Lists typicode todos",
-    description: "List typicode REST sample data",
-    type: "command",
-    handler: __dirname + "/TypicodeTodos.handler",
-    options: [
-        {
-            name: "id",
-            description: "The todo to list",
-            type: "number"
-        }
-    ]
-};
+This tutorial creates the following command in Zowe CLI:
+
+```
+zowe zowe-cli-sample list typicode-todos`
 ```
-This describes the syntax of your command.
+
+#### Describing the syntax of your command **[correct?]**
+
+1. Navigate to `zowe-cli-sample-plugin/src/cli/list` and create a folder titled `typicode-todos`. 
+
+2. In this folder, create a file named `TypicodeTodos.definition.ts`.
+
+    Edit the `TypicodeTodos.definition.ts` file to contain the following code:
+
+    ```typescript
+    import { ICommandDefinition } from "@zowe/imperative";
+    export const TypicodeTodosDefinition: ICommandDefinition = {
+        name: "typicode-todos",
+        aliases: ["td"],
+        summary: "Lists typicode todos",
+        description: "List typicode REST sample data",
+        type: "command",
+        handler: __dirname + "/TypicodeTodos.handler",
+        options: [
+            {
+                name: "id",
+                description: "The todo to list",
+                type: "number"
+            }
+        ]
+    };
+    ```
+
+    The `TypicodeTodos.definition.ts` file describes the syntax of your command.
 
 #### Defining command to list group
 
-Within the file `zowe-cli-sample-plugin/src/cli/list/List.definition.ts`, add the following code below other `import` statements near the top of the file:
-```typescript
-import { TypicodeTodosDefinition } from "./typicode-todos/TypicodeTodos.definition";
-```
+1. In `zowe-cli-sample-plugin/src/cli/list/List.definition.ts`, add the following code below other `import` statements near the top of the file:
+    ```typescript
+    import { TypicodeTodosDefinition } from "./typicode-todos/TypicodeTodos.definition";
+    ```
 
-Then add `TypicodeTodosDefinition` to the children array. For example:
-```
-children: [DirectoryContentsDefinition, TypicodeTodosDefinition]
-```
+2. To the children array, add `TypicodeTodosDefinition`.
+
+    For example:
+    ```
+    children: [DirectoryContentsDefinition, TypicodeTodosDefinition]
+    ```
+    The command is defined in the list group. **[correct?]**
```

Comment:
@adam-wolfe please respond

Reply:
I would suggest something like: "The command is added to the `list` command group."